src/workbench.ts

```diff
@@ -0,0 +1,51 @@
+import './missing-services'
+import { IAnyWorkspaceIdentifier, UNKNOWN_EMPTY_WINDOW_WORKSPACE } from 'vs/platform/workspace/common/workspace'
+import { IWorkspace } from 'vs/workbench/services/host/browser/browserHostService'
+import { IWorkbenchConstructionOptions } from 'vs/workbench/browser/web.api'
+import { isFolderToOpen, isWorkspaceToOpen } from 'vs/platform/window/common/window'
+import { getSingleFolderWorkspaceIdentifier, getWorkspaceIdentifier as getWorkspaceIdentifierFromUri } from 'vs/workbench/services/workspaces/browser/workspaces'
+import { URI } from 'vs/base/common/uri'
+import { toLocalISOString } from 'vs/base/common/date'
+
+let workbenchContainer: HTMLElement = document.body
+let workbenchConstructionOptions: IWorkbenchConstructionOptions = {}
+let workspaceIdentifier: IAnyWorkspaceIdentifier = UNKNOWN_EMPTY_WINDOW_WORKSPACE
```

Comment:
When we call `reinitializeWorkspace` in `configurationServiceOverride`, do we need to change this as well? Not sure if it's used after initialization.

Reply:
It's only used at initialization